app-starter/static/app-conf.json

```
@@ -229,7 +229,8 @@
     },
     "wgu-attributetable": {
       "target": "menu",
-      "win": true
+      "win": true,
+      "activateTableMapInteraction": true
```

I'd prefer something like

```suggestion
      "syncTableMapSelection": true
```